screenpy/actions/silently.py

```diff
@@ -89,8 +89,16 @@ def __init__(self, duck: Resolvable):
         self.duck = duck
 
 
-T_duck: TypeAlias = Union[Performable, Answerable, Resolvable]
-T_quack: TypeAlias = Union[SilentlyAnswerable, SilentlyPerformable, SilentlyResolvable]
+T_duck: TypeAlias = Union[
+    Answerable,
+    Performable,
+    Resolvable,
+]
+T_silent_duck: TypeAlias = Union[
```

Comment:
I still vote for `T_grey_duck`.

Reply:
My concern with that was non-North American folks may not get the reference. Even if they did, would it be confusing?

Reply:
Minnesotans and Swedes will get the joke. :P

But yes, i vote for understandable over funny.

Reply:
Oh, I tried to make it work.  It was right there...